RN - Core.DataMinerSystem - 1.1.1.12 (#3354)
SkylineCommunications/dataminer-docs: release-notes/Code_Libraries/Skyline.DataMiner.Core/Skyline.DataMiner.Core.DataMinerSystem/Skyline_DataMiner_Core_DataMinerSystem_Range_1.1.md

> [!NOTE]
> Range 1.1.x.x is supported as from **DataMiner 10.1.11**. It makes use of a change introduced in DataMiner 10.1.11 that makes it possible to obtain table cell data using the primary key. In earlier DataMiner versions, the display key was needed to obtain this data.

### 1.1.1.12

#### Fix - Breaking change reverted for InterAppCalls

> When using the Skyline.DataMiner.Core.InterAppCalls.Common NuGet package and versions 1.1.1.10/1.1.1.11 of the DataMinerSystem package will cause run-time errors. This breaking change has been reverted.

Reviewer: ```suggestion
When the Skyline.DataMiner.Core.InterAppCalls.Common NuGet package was used, versions 1.1.1.10/1.1.1.11 of the DataMinerSystem package could cause run-time errors. This breaking change has been reverted.
```
